modules/codemod/lib/v7/renameLayoutImports.ts

```diff
@@ -79,12 +79,14 @@ const transform: Transform = (file, api) => {
     });
   } else {
     // create new import
-    foundImport[0].insertBefore(
-      j.importDeclaration(
-        reactLayoutSpecifiers.map(mapToSpecifiers),
-        j.stringLiteral('@workday/canvas-kit-react/layout')
-      )
-    );
+    if (foundImport.length) {
```

Comment:
The codemod assumed an import would always be found, but it wouldn't if the file under the codemod didn't import from `Layout` which caused the codemod to crash.

Reply:
ahhhh I also have this PR update this codemod but we can merge this in first! https://github.com/Workday/canvas-kit/pull/1567